config/authorization/config.ex

```diff
@@ -70,6 +33,7 @@ defmodule Acl.UserGroups.Config do
                         "http://schema.org/ContactPoint",                          
                         "http://data.lblod.info/vocabularies/leidinggevenden/Bestuursfunctie",
                         "http://data.lblod.info/vocabularies/erediensten/CentraalBestuurVanDeEredienst",
+                        "http://data.lblod.info/vocabularies/erediensten/CentraleBestuursorgaan",
```

Comment:
We're not using this type. Why is it back?

Reply:
in your previous change you let it in one side and removed it in the other :P I saw the diff so I took the most complete array. I'll fix it

Reply:
yes, too many places to put the same information.

Reply:
I removed it, now there is just one place to put the resource type to avoid the confusion in the future